README.md

```diff
@@ -46,26 +49,54 @@ To push to Heroku, go to [How to Upload Changes to Heroku](https://github.com/op
 
 ### Running Docker Container Locally ###
 
-When running the docker container locally for debugging, it’s connected to the same Elasticsearch and Maria instances as Heroku.
+You can run a full instance of Grimoire locally, or, if you have the credentials, you can debug the production instance by connecting to the same elasticsearch and Maria instances as Heroku.
 
-If you want to run the project on your local computer as opposed to on Heroku, use this command **after stopping the container running on Heroku**:
+#### Debug remote instance
 
-```
-docker run --env DB_HOST --env DB_NAME --env DB_PASS && \
---env DB_USER --env ELASTIC_URL --env GITHUB_KEY && \
---env KIBANA_URL --env SLACK_KEY test_dockerfile
-```
-If you want to run the container locally, but use local files for testing purposes, you can use a command like this:
+You can run the docker container locally for debugging, and if you have the credentials, connect to the same Elasticsearch and Maria instances as Heroku.
 
-```
-docker run -v $(pwd)/credentials.cfg:/override.cfg && \
--v $(pwd)/infra.cfg:/infra.cfg -v $(pwd)/dashboard.cfg:/dashboard.cfg && \
--v $(pwd)/logs:/logs -v $(pwd)/identities.yaml:/identities.yaml && \
--v $(pwd)/projects.json:/projects.json  && \
--v $(pwd)/aliases.json:/aliases.json grimoirelab/installed
-```
+1. **Stop the container running on Heroku.**
+1. Run this command to run the generated `test_dockerfile`
+
+    ```
+    docker run --env DB_HOST --env DB_NAME --env DB_PASS && \
+    --env DB_USER --env ELASTIC_URL --env GITHUB_KEY && \
+    --env KIBANA_URL --env SLACK_KEY test_dockerfile
+    ```
+
+Alternatively, if you want to run the container locally, but use local files to test their effects before pushing to github, you can:
+
+1. Stop the production container on Heroku.
+1. Run `set_config.py` and `create_identities.py` to generate the `credentials.cfg` and `identities.yaml` files, respectively.
+1. Run `make start.remote` to run the docker container.
+1. Run `make logs.remote` to watch the container logs. Once Sir Mordred is running, you can also watch `logs/all.log`.
+1. Run `make help` to see the other `*.remote` commands.
+
+Although the container in this command is *grimoirelab/installed* as opposed to the *test_dockerfile* image used for the Heroku app, it essentially completes the same job, with the exception of not running the extra scripts that I have written.
+
+#### Run full local instance
+
+You can run a full grimoirelab instance locally.
+
+1. Copy `devstack/override.cfg.sample` file into `devstack/override.cfg`.
```

Comment:
@pomegranited, I started testing this but for the life of me could not find a devstack directory in this repository.  Tried https://github.com/chaoss/grimoirelab itself, not there either.